observatory-platform/observatory/platform/workflows/templates/doc.md.jinja2

```diff
@@ -1,3 +1,70 @@
-# {{ workflow_name }}
+{# Copyright 2021 Curtin University
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# Author: Aniek Roelofs -#}
+# {{ workflow_class }}
 
 Add Workflow Description here
+
+ ```eval_rst
++------------------------------+--------------------------------------+
```

Comment:
I like the idea of having a summary table like this, but thinking about it a bit more, I am wondering if some of these fields are too specific to the COKI setup, and also to current releases.

Specifically:
- Average runtime will depend on the computational resources people throw at it to some degree.
- Average download size depends on the release. Datasets are expected to grow in size over time, and we don't necessarily want to keep updating this.
- Harvest frequency/update frequency. I'm a bit unclear with the terminology.  Is one of them how often the dataset is updated by the provider, and how often it gets scheduled as a dagrun?  Might need to make those names more explicit if that is the case.

So maybe the second column should have "Default: " in front of many of the entries to indicate that is the default setting and indicate it is configurable.

Reply:
Yeah, thanks for the feedback, it could use some improvements I think.

I agree the runtime/download size can be quite variable, but I think it would still be good to have a rough estimation. The thing that I would like to convey is when you are not familiar with the workflow or don't have access to Airflow to get an idea on how big the data is and how long it will take to run the workflow.
Just to get a rough idea whether it is a workflow that will take 24+ hours and the resulting table is multiple GBs or whether it will be done in 15 mins and the table is just a couple of MBs.
It wasn't the intention to keep updating these, but just give an average.

Maybe as an alternative we could create some categories/bins, like: '< 1 hour', '1-6 hours', '6-12 hours', 12-24 hours', '>24 hours' and similar for size.
What do you think? Do you have any other ideas?

Yeah the harvest frequency/update frequency is like you said, I wanted to include both how often the data is updated by the provider and how often the workflow updates the data.
Maybe 'Workflow update frequency' and 'Provider update frequency'? 
I don't want to make the names too long, any suggestions?